.github/workflows/kuksa_databroker_build.yml

```diff
@@ -78,31 +78,13 @@ jobs:
           crate: cargo-strip
           version: latest
 
-      - uses: actions-rs/install@v0.1
-        with:
-          crate: cargo2junit
-          version: latest
-
-      - name: Run Tests
+      - name: Show toolchain information
         working-directory: ${{github.workspace}}
         run: |
-          cargo test --all-targets -- -Z unstable-options --report-time --format json | cargo2junit > results.xml;
```

Comment:
If we remove the "unstable part" we cannot do the result upload part. less important as I see it as the log anyway is visible in stdout.
But if we do not have the report part there is no need to have it as tests anyway are run by "cargo tarpaulin" below